css/style.css

```diff
@@ -5,19 +5,21 @@ body {
 }
 
 #header {
+  padding-top: 1rem;
   width: 100%;
   height: 100%;
   display: flex;
   justify-content: space-between;
   background-color: gray;
+  align-items: center;
 }
 
 .nav-link {
   text-decoration: none;
   color: white;
   list-style: none;
   display: flex;
-  padding: 0.5em;
```

Comment:
Invalid value for `padding`        csstree/validator
Unexpected whitespace before ";"   declaration-block-semicolon-space-before

Add  a value between padding: and ;
example: padding: 1em;